packages/common/src/ConfigResponse.ts

```diff
@@ -21,8 +21,14 @@ export interface ConfigEntry {
   gsnConfig: Partial<GSNConfig>
 }
 
+export interface BlacklistedRelayServerInfo {
```

Comment:
I think it should be a `string[]`, not some compex structure, just like the preferred relays

Reply:
Don't you think we should differentiate between manager address and host? Just drop everything in one string array?

Reply:
Fixed.